jet/wordcount-compute-isolation/pom.xml

```diff
@@ -84,5 +84,7 @@
         <!-- needed for checkstyle/findbugs -->
         <main.basedir>${project.parent.parent.basedir}</main.basedir>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<!-- TODO Once Hazelcast v5.5 is released, can become non-SNAPSHOT -->
+		<hazelcast.version>5.5.0-SNAPSHOT</hazelcast.version>
```

Comment:
How do we track this ?
In the meantime, does code-samples compile fully?
I presume SNAPSHOT is required for some changes as 5.4.0 doesn't have the change?

Reply:
> How do we track this ?

I don't know, I have no mechanism for enforcing this.

> In the meantime, does code-samples compile fully?

Yes - the PR builder is passing.
This project is using `5.5.0-SNAPSHOT` of EE, which is available in the repo so it's ok.

> I presume SNAPSHOT is required for some changes as 5.4.0 doesn't have the change?

This is demonstrating new 5.5 functionality.
It's _likely_ newer functionality will be EE centric so we shouldn't really have many problems.

Reply:
thanks